docs/revsets.md

```diff
@@ -58,14 +58,14 @@ only symbols.
 * `~x`: Revisions that are not in `x`.
 * `x-`: Parents of `x`.
 * `x+`: Children of `x`.
-* `:x`: Ancestors of `x`, including the commits in `x` itself.
-* `x:`: Descendants of `x`, including the commits in `x` itself.
-* `x:y`: Descendants of `x` that are also ancestors of `y`. Equivalent to
-  `x: & :y`. This is what `git log` calls `--ancestry-path x..y`.
+* `::x` or `:x`: Ancestors of `x`, including the commits in `x` itself.
```

Comment:
Good idea.  I made that change and also said we'll delete it in 0.15+ instead of 0.12+, giving users ~6 months assuming we'll do a ~monthly release from now on as we've talked about.